modules/light-clients/07-tendermint/proposal_handle_test.go

```diff
@@ -125,39 +125,39 @@ func (suite *TendermintTestSuite) TestCheckSubstituteAndUpdateState() {
 			subjectClientStore := suite.chainA.App.GetIBCKeeper().ClientKeeper.ClientStore(suite.chainA.GetContext(), subjectPath.EndpointA.ClientID)
 			substituteClientStore := suite.chainA.App.GetIBCKeeper().ClientKeeper.ClientStore(suite.chainA.GetContext(), substitutePath.EndpointA.ClientID)
 
-			expectedConsState := substitutePath.EndpointA.GetConsensusState(substituteClientState.GetLatestHeight())
-			expectedProcessedTime, found := ibctm.GetProcessedTime(substituteClientStore, substituteClientState.GetLatestHeight())
+			expectedConsState := substitutePath.EndpointA.GetConsensusState(substituteClientState.LatestHeight)
```

Comment:
`expPass` is not used in this test

Reply:
https://github.com/cosmos/ibc-go/issues/6001